drivers/include/isl29125.h

```diff
@@ -115,13 +146,23 @@ typedef struct {
  * @param[in] mode          operation mode
  * @param[in] range         measurement range
  * @param[in] resolution    AD conversion resolution
+ * @param[in] interrupt_status          Interrupt status
+ * @param[in] interrupt_persist         Interrupt persistency
+ * @param[in] interrupt_conven          RGB conversion done to interrupt control, enable
+ * @param[in] lower_threshold           Lower interrupt threshold
+ * @param[in] upper_threshold           Upper interrupt threshold
```

Comment:
here it's called `upper_threashold`, but in the implementationyou used `higher_threshold`